src/app/frontend/logs/logs_controller.js

```diff
@@ -0,0 +1,41 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Controller for the logs view.
+ *
+ * @final
+ */
+export default class LogsController {
+  /**
+   * @param {!backendApi.Logs} podLogs
+   * @param {!Object} logsColorInversionService
```

Comment:
When I use like this I have syntax error because of ```.```
In constructor
```    /** @private {!./logs_service.LogColorInversionService} */
    this.logsColorInversionService_ = logsColorInversionService;
``` there is no errors but for ```@params``` I can not use like this